Problem: find-three-consecutive-integers-that-sum-to-a-given-number
Submission ID: 1695692673
Submission Time: 2025-07-12 15:03:28
============================================================

State: SUCCESS
Status: Accepted
Language: Python

Runtime: 0 ms
Memory: 12.36 MB

Test Cases: 379/379 passed

Runtime Percentile: 100%
Memory Percentile: 81.17639999999999%

============================================================
RAW SUBMISSION DATA
============================================================
{
  "status_code": 10,
  "lang": "python",
  "run_success": true,
  "status_runtime": "0 ms",
  "memory": 12360000,
  "display_runtime": "0",
  "question_id": "2278",
  "elapsed_time": 45,
  "compare_result": "1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111",
  "code_output": "",
  "std_output": "",
  "last_testcase": "",
  "expected_output": "",
  "task_finish_time": 1752357807574,
  "task_name": "judger.judgetask.Judge",
  "finished": true,
  "total_correct": 379,
  "total_testcases": 379,
  "runtime_percentile": 100,
  "status_memory": "12.4 MB",
  "memory_percentile": 81.17639999999999,
  "pretty_lang": "Python",
  "submission_id": "1695692673",
  "status_msg": "Accepted",
  "state": "SUCCESS"
}